code/game/machinery/atmos_exporter.dm

```diff
@@ -0,0 +1,269 @@
+/obj/machinery/atmos_points
+	name = "atmospheric export computer"
+	desc = "used to monitor the galactic atmos markets"

```

Comment:
Used to monitor the GAM.

then in the monitor it should have a sexy background UI with GALACTIC ATMOSPHERICS MARKET written it with the entire machine's casing having GAM written around it